.github/workflows/publish_docs.yml

```diff
@@ -0,0 +1,46 @@
+name: publish docs to github pages
+# taken from https://github.com/peaceiris/actions-gh-pages#%EF%B8%8F-static-site-generators-with-python
+
+on:
+  push:
+    branches:
+      - mjj/sphinx_docs
```

Comment:
This should point to Allen-Synthesis/EuroPi:main (I don't know the exact syntax)